### PR TITLE
refactor(ivy): refactor getBeforeNodeForView for readability

### PR DESCRIPTION
**Pull request by pkozlowski-opensource**
Minor cleanup done while going over the code...

### CODE REVIEW

File: packages/core/src/render3/node_manipulation.ts [hunk 1]
```diff
@@ -729,15 +729,14 @@ function getHighestElementOrICUContainer(tNode: TNode): TNode {
...
 
-export function getBeforeNodeForView(index: number, lContainer: LContainer) {
-  const containerNative = lContainer[NATIVE];
-
-  if (index + 1 < lContainer.length - CONTAINER_HEADER_OFFSET) {
-    const view = lContainer[CONTAINER_HEADER_OFFSET + index + 1] as LView;
+export function getBeforeNodeForView(viewIndexInContainer: number, lContainer: LContainer): RNode {
+  const nextViewIndex = CONTAINER_HEADER_OFFSET + viewIndexInContainer + 1;
+  if (nextViewIndex < lContainer.length) {
+    const view = lContainer[nextViewIndex] as LView;
```
Comment (alfaproject) [inline packages/core/src/render3/node_manipulation.ts]: I guess `lView` is more grep-able and being used everywhere, right?

File: packages/core/src/render3/node_manipulation.ts [hunk 2]
```diff
@@ -729,15 +729,14 @@ function getHighestElementOrICUContainer(tNode: TNode): TNode {
...
-
-  if (index + 1 < lContainer.length - CONTAINER_HEADER_OFFSET) {
-    const view = lContainer[CONTAINER_HEADER_OFFSET + index + 1] as LView;
-    const viewTNode = view[T_HOST] as TViewNode;
-    return viewTNode.child ? getNativeByTNode(viewTNode.child, view) : containerNative;
+export function getBeforeNodeForView(viewIndexInContainer: number, lContainer: LContainer): RNode {
+  const nextViewIndex = CONTAINER_HEADER_OFFSET + viewIndexInContainer + 1;
+  if (nextViewIndex < lContainer.length) {
+    const lView = lContainer[nextViewIndex] as LView;
+    const viewTNode = lView[T_HOST] as TViewNode;
```
Comment (petebacondarwin) [inline packages/core/src/render3/node_manipulation.ts]: why not `tViewNode` keeping in line with the type?

**Review by petebacondarwin — APPROVED** [review]
Always love to see tidy ups ❤️

**Review by ocombe — APPROVED** [review]

**Review by mhevery — APPROVED** [review]

**Comment by mhevery** [comment]
http://test/OCL:249635137:BASE:249635166:1558620165020:ac601484

**Comment by pkozlowski-opensource** [comment]
merge-assistance: looks like presubmit was run by @mhevery but PR status was not updated...

**Comment by AndrewKushnir** [comment]
@pkozlowski-opensource there were some flaky tests, I've forced g3 status to make this PR "green". Thank you.

**Pull request closed without merging**